zkevm-circuits/src/bytecode_circuit/bytecode_unroller.rs

```diff
@@ -786,6 +877,9 @@ mod tests {
     /// Verify unrolling code
     #[test]
     fn bytecode_unrolling() {
+        use eth_types::Bytecode;
+        use halo2_proofs::halo2curves::bn256::Fr;
```

Comment:
No sense to have imports local on each test. Please put all of them in common within the top level module imports.

Reply:
This is because moving import outside the method will meet `unused import` error when using make clippy.
As below.
<img width="632" alt="image" src="https://user-images.githubusercontent.com/108982045/212529338-f792b159-3397-47d9-b11e-6ddf0b03f9f9.png">

Reply:
> This is because moving import outside the method will meet `unused import` error when using make clippy. (As below.)

I resolve this by adding `#![allow(unused_imports)]` to the mod.

It's a little weird.